### PR TITLE
Noita: Add the useful classification to important perks, making them progression + useful

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
Makes it so the Immunity perks, All-Seeing Eye, and Tinker with Wands Everywhere are Progression and Useful.
They are the main thing a player would receive and be like "woah, the good item!!!"
Spatial Awareness doesn't get this classification because it is only really used for one check.

## How was this tested?
Test gen

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

File: worlds/noita/items.py
Comment (Exempt-Medic) [inline worlds/noita/items.py:1]: These lines are now over 120 characters

**Review by Exempt-Medic — APPROVED** [review]
Changes LGTM. Tested by generating a game, hosting it, releasing it and looking at the items using the TextClient (they had both flags).

**Review by nicholassaylor — APPROVED** [review]
LGTM

**Comment by NewSoupVi** [comment]
I'm merging this because I want this case to be more likely to come up in RC2 in case some clients are "doing it wrong", and this seems like a pretty simple / non-destructive PR that can be reverted if necessary

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**